src/bindings/c/include/openvino/c/ov_compiled_model.h

```diff
@@ -71,11 +71,11 @@ ov_compiled_model_create_infer_request(const ov_compiled_model_t* compiled_model
  * @brief Sets properties for the current compiled model.
  * @ingroup compiled_model
  * @param compiled_model A pointer to the ov_compiled_model_t.
- * @param property ov_property_t.
+ * @param property ov_properties_t.
  * @return Status code of the operation: OK(0) for success.
  */
 OPENVINO_C_API(ov_status_e)
-ov_compiled_model_set_property(const ov_compiled_model_t* compiled_model, const ov_property_t* property);
+ov_compiled_model_set_property(const ov_compiled_model_t* compiled_model, const ov_properties_t* property);
```

Comment:
yes, C does support variable parameter number.

Reply:
Latest implement adopts property array, do we need change to variable number of arguments?

Reply:
I think it will be great and it will be more aligned with C++ OV 2.0 API, which allows to have next constructions:
```
core.set_property("DEV_NAME",  ov::num_streams(8), ov::enable_profiling(true));
```

Reply:
If we have this approach maybe we don't need to have properties list in C API